app/controllers/enrollments_controller.rb

```diff
@@ -15,8 +16,8 @@ def create
   end
 
   def discard
-    @enrolled_event = Enrollment.find_by(user_id: @current_user[:id], event_id: params[:eventid], created: false)
-    @enrolled_event.destroy
+    enrolled_event = Enrollment.find_by(user_id: @current_user[:id], event_id: params[:eventid], created: false)
```

Comment:
Can we use like this:  @current_user.enrollments.where(event_id: params[:eventid])